.github/workflows/windows-build.yml

```
@@ -3,37 +3,27 @@ on:
     branches:
       - main
       - fix-exe
```

Let's temporarily add testingv4artifact branch to the workflow, to test it.
```suggestion
      - fix-exe
      - testingv4artifact
```